cid-redirects.json

```diff
@@ -268,8 +268,8 @@
   "/03Send-Data/Sources/04Reference-Information-for-Sources/Source-Log-Encoding": "/docs/send-data/reference-information/source-log-encoding",
   "/03Send-Data/Sources/04Reference-Information-for-Sources/Timestamps,-Time-Zones,-Time-Ranges,-and-Date-Formats": "/docs/send-data/reference-information/time-reference",
   "/03Send-Data/Sources/04Reference-Information-for-Sources/Using-Wildcards-in-Paths": "/docs/send-data/reference-information/use-wildcards-paths",
-  "/03Send-Data/Sumo_Logic_Distribution_for_OpenTelemetry": "/docs/send-data/sumo-distribution-opentelemetry",
-  "/03Send-Data/Sumo_Logic_OpenTelemetry_Distribution": "/docs/send-data/sumo-distribution-opentelemetry",
+  "/03Send-Data/Sumo_Logic_Distribution_for_OpenTelemetry": "/docs/send-data/sumo-logic-opentelemetry",
+  "/03Send-Data/Sumo_Logic_OpenTelemetry_Distribution": "/docs/send-data/sumo-logic-opentelemetry",
```

Comment:
I think we should make it more explicit that this is about the `OpenTelemetry Collector` because there are also `OpenTelemetry Libraries` (aka instrumentation) and we have our distributions for both of those.

And we'd need to have documentation on instrumentation if we don't have it already (we probably do and it's somewhere around tracing but I believe we should link to them somewhere here as well).

BTW the legally correct name is `Sumo Logic Distribution for OpenTelemetry Collector`.

Reply:
Thank you!! Made some fixes to address this in this next round of editing. Let me know if this works: https://github.com/SumoLogic/sumologic-documentation/pull/1617/files.